platform/golden/provisioners/roles/override_kernel_parameters/files/dropin-profile-kvm-for-grub.cfg

```diff
@@ -0,0 +1,2 @@
+GRUB_CMDLINE_LINUX="$GRUB_CMDLINE_LINUX modprobe.blacklist=nouveau pci=realloc=on pcie_ports=native usbcore.nousb vsyscall=none intel_iommu=on,sm_on amd_iommu=on amd_iommu_intr=vapic iommu=pt iommu.strict=1 iommu.forcedac=1 kvm-amd.avic=1 kvm-amd.nested=0 kvm-amd.npt=1 transparent_hugepage=never nmi_watchdog=0 default_hugepagesz=2M hugepagesz=2M hugepages=400000 video=efifb:off iommu.passthrough=1 iommu.forcedac=1 rd.driver.pre=vfio-pci pcie_port_pm=off pcie_bus_perf pcie_aspm=off"
```

Comment:
I read yesterday hugepages is incompatible with `HMM`. WIll need to test `HMM` to see if it produces anyvalue.

Reply:
Would make sense to make a dropin file for amd, and a dropin for intel, and a dropin for all cases.